dap_aria_mapping/pipeline/data_collection/patents_citations.py

```diff
@@ -0,0 +1,268 @@
+"""
+patent citations pipeline
+--------------------------
+A pipeline that collects the citing works (via forward and backward citations)
+for each patent collected using pipeline/data_collection/patents.py.
+
+python dap_aria_mapping/pipeline/data_collection/patents_citations.py run
+
+NOTE: If the flow is in production, it will cost money to run this flow.
+DO NOT run this flow unless you have explicitly confirmed with Genna or India.
+"""
+from typing import List, Dict, Tuple
+
+from metaflow import FlowSpec, step, Parameter
+import pandas as pd
+from google.cloud import bigquery
+
+from dap_aria_mapping.utils.patents import (
+    format_list_of_strings,
+    chunk,
+)
+
+from dap_aria_mapping.utils.conn import est_conn
+from dap_aria_mapping import logger
+
+# 15 bigquery chunks to run in parallel
+CHUNK_SIZE = 15
+
+# early year time point
+EARLY_YEAR = 2007
+
+# late year time point
+LATE_YEAR = 2017
+
+# create connection to bigquery
+conn = est_conn()
+
+
+def patents_backward_citation_query() -> str:
+    """Returns a query that collects patent citation information
+        (i.e. 'patent citations' in Google Patents front end)
+        for a list of focal_ids.
+
+    Returns:
+        str: Base query string.
+    """
+    return "SELECT p.publication_number as focal_id, c.publication_number as patent_citation from `patents-public-data.patents.publications` as p, unnest(citation) c WHERE p.publication_number IN ({})"
```

Comment:
Minor, but might be more readable to break up long SQL query strings into multiline strings with `"""` or `()`